srcpkgs/dolphin-plugins/template

```diff
@@ -1,11 +1,10 @@
 # Template file for 'dolphin-plugins'
 pkgname=dolphin-plugins
 version=20.04.3
-revision=1
+revision=2
 build_style=cmake
 hostmakedepends="extra-cmake-modules gettext qt5-qmake qt5-host-tools kcoreaddons kconfig"
 makedepends="qt5-devel kxmlgui-devel kio-devel ktexteditor-devel dolphin"
-depends="dolphin"
```

Comment:
You don't need to do any changes to dolphin-plugins. That package was correct before.